.github/renovate.json5

```diff
@@ -1,20 +1,19 @@
 {
   $schema: 'https://docs.renovatebot.com/renovate-schema.json',
   extends: [
-    'config:base',
-    ':dependencyDashboard',
+    'config:recommended',
     ':labels(dependencies)',
     ':maintainLockFilesMonthly', // update non-direct dependencies monthly
     ':prConcurrentLimit10', // only 10 open PRs at the same time
```

Comment:
Could you remove this?

As there are some PRs we cannot merge easily (due to the Renovate bug above, or needing more work/testing), we would prefer not have a concurrency limit, but still have a rate-limit

Reply:
Instead of removing the line, I've replaced it with the `:prConcurrentLimitNone` preset. I've added the `:prHourlyLimit2` (which is the default setting!) to make the config easier to understand.